mpdev/internal/tf/overwrite_test.go

```diff
@@ -406,3 +508,123 @@ spec:
       description: The image name for the disk for the VM instance.
       varType: string
 `
+
+var metadataDisplayWithEnumsSingle string = `
+spec:
+  ui:
+    input:
+      variables:
+        source_image:
+          name: source_image
+          title: Source Image
+          enumValueLabels:
+            [
+              {
+                label: 'wordpress-1',
+                value: 'projects/click-to-deploy-images/global/images/wordpress-1'
+              }
+            ]
+          xGoogleProperty:
+            type: ET_GCE_DISK_IMAGE
+`
+
+var metadataDisplayWithEnumsSingleReplaced string = `
+spec:
+  ui:
+    input:
+      variables:
+        source_image:
+          name: source_image
+          title: Source Image
+          enumValueLabels:
```

Comment:
nit: For consistency I'd keep this as all normal yaml
enumValueLabels:
  - label:
    value: